pkg/agent/storage/keyvalue_store_ci.go

```diff
@@ -0,0 +1,37 @@
+package storage
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/go-kit/kit/log"
+	"github.com/kolide/launcher/pkg/agent/types"
+	"github.com/stretchr/testify/require"
+	"go.etcd.io/bbolt"
+)
+
+const (
+	dbTestFileName = "test.db"
+)
+
+func NewCIKeyValueStore(t *testing.T, logger log.Logger, bucketName string) types.KVStore {
+	if os.Getenv("CI") == "true" {
```

Comment:
Not sure how folks feel about this - we would run against bbolt locally, but forgo bbolt in GitHub CI. There's probably an improvement here to be made, so we don't accidentally regress something in the bbolt implementation and not notice it in the CI runs.

Reply:
I've seen it be done, like in the fyne codebase they have some files with
```
//go:build ci
// +build ci
```
It doesn't look like we've done that anywhere, I assume that will require some changes to make to pipe that build var down.

Reply:
That looks like it's adding in a `ci` build tag. Can we use the existing `test` build tag that's implicit for the test suite? Partly, if feels very weird pulling `testing` in to prod builds

Reply:
Eh, might not ultimately be worth it.

You know... If this was in it's own `pkg` it wouldn't get linked into production builds.